Sources/Utils/Extensions/AVAsset+Extensions.swift

```diff
@@ -55,15 +55,15 @@ extension AVAsset {
 
   var g_videoDescription: CMFormatDescription? {
     if let object = tracks(withMediaType: AVMediaTypeVideo).first?.formatDescriptions.first {
-      return object as! CMFormatDescription
+      return (object as! CMFormatDescription)
     }
```

Comment:
Perhaps we could refactor this method to do the casting in the if statement so that we get rid of the force unwrapping. What do you think?

```swift
guard let object = tracks(withMediaType: AVMediaTypeVideo).first?.formatDescriptions.first as? CMFormatDescription else {
  return nil
}

return object
```

Reply:
I've changed 'if let' to 'guard let', but it's just a syntax sugar - Conditional downcast to CoreFoundation type 'CMFormatDescription' will always succeed :) 'guard' let looks better